`cargo build --release` failing due to dependency using `#![feature]`

When building from main I am getting -


```
   Compiling to_snake_case v0.1.1
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> /Users/mrschre/.cargo/registry/src/index.crates.io-6f17d22bba15001f/to_snake_case-0.1.1/src/lib.rs:31:1
   |
31 | #![feature(let_chains)]
   | ^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `to_snake_case` (lib) due to 1 previous error
```

Rust version is rustc 1.77.2 (25ef9e3d8 2024-04-09)

Is it intended that the project be built with the rust nightly build?

--- a/.gitignore
+++ b/.gitignore
@@ -5,3 +5,5 @@ target/
 **/Cargo.lock
 
 rustc-ice*
+
+.idea

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,0 +1,2 @@
+[toolchain]
+channel = "nightly"

